pandas/tests/frame/test_missing.py

```diff
@@ -388,6 +388,33 @@ def test_frame_fillna_limit(self):
         expected.values[:3] = np.nan
         tm.assert_frame_equal(result, expected)
 
+    def test_frame_fillna_axis(self):
+        # GH 17399
+        # with limit
+        df = DataFrame(np.random.randn(10, 4))
+        df.iloc[1:4, 1:4] = nan
+        expected = df.copy()
+        expected.iloc[1:4, 1:3] = 0
+        result = df.fillna(value=0, limit=2, axis=1)
+        tm.assert_frame_equal(result, expected)
+        # with no limit
```

Comment:
Style nit: could you add a blank line between tests, or break them into separate tests?